views/firstLogin/sections/thanks.hbs

```diff
@@ -2,14 +2,14 @@
     <div class="welcomeGreeting">
         <div class="welcomeGreetingImage text-center" id="welcome">
             <div class="content">
-                <img src="/images/firstLogin/welcome_screen.svg">
+                <img alt="{{$t "first_login.img_alt.wavingStudents"}}" src="/images/firstLogin/welcome_screen.svg">
                 <hr class="mt-0">
                     <h1 class="h2">{{$t "firstLogin.welcome.text.welcomeTo" (dict "shortTitle" @root.theme.short_title)}}</h1>
                 <hr>
                 <!-- fokus here -->
-                <a href=""></a> 
+                <a aria-hidden="true" href=""></a>
```

Comment:
Why is the link hidden?

Reply:
I got rid of the empty a tag and I think the arrow down is a nice to have but not necessary for blind people as it just scrolls you down to the content, what is done by tabbing anyways..